### ISSUE TITLE
Dropbox Token expires every some time.

### ISSUE THREAD

**Issue by sparshy** [johannesjo/super-productivity#1977]
### Your Environment

- Version used: 7.10.1 Used Appimage 
- Operating System and version: Linux Ubuntu 18.04
- Desktop Environment: Gnome
- Browser Name and version: Browser Chrome 98.0

### Expected Behavior
If the token has expired it should refresh i think.

### Current Behavior
![image](https://user-images.githubusercontent.com/8553391/159837245-81b773c7-be90-4d68-9814-3ab93ff0c5e0.png)

### Steps to Reproduce (for bugs)

Does the error happen once or are you able to reproduce it reliably?  Reliably, Happens on Both Linux and Android

1. Configured Dropbox. 
2. Works fine for some time.
3. After that I see the above error message

### Console Output

![image](https://user-images.githubusercontent.com/8553391/159837557-d00bb3a4-6f95-483b-ba74-ae488e32f584.png)

![image](https://user-images.githubusercontent.com/8553391/159837777-476490c7-1164-47b5-acaf-4c660842611a.png)

**Comment by johannesjo** [comment]
Thanks for opening this up. Dropbox changed their policy on tokens, but a fix is already on the way. You can use it with the latest edge release or you wait for the next regular release.

**Comment by sparshy** [comment]
Thank you for the reply @johannesjo. Is it possible to get the edge release in form of AppImage? Like which branch corresponds to snaps edge release?

**Comment by johannesjo** [comment]
@sparshy yes, but you would have to build the app for yourself. It's the master branch. You can check it out and use the instructions in the readme.

**Comment by clarkdowding** [comment]
I'm also experiencing this bug. Looking forward to the fix. Thanks!

**Comment by johannesjo** [comment]
The fix should be out now. Please let me know if it works out for you!

**Comment by clarkdowding** [comment]
> The fix should be out now. Please let me know

I updated two Linux machines and one Max yesterday to 7.11.5 and all three devices are syncing constantly to my DropBox account. Prior to 7.11.5 I was getting the error pop-up shown by @sparshy in this bug report constantly. I kept re-authorizing DropBox. Immediately after upgrading to 7.11.5 I had to re-authorize once more on each machine and then no more.

I am using SuperProductivity very heavily right now because I'm migrating my personal and professional tasks into it from Things 3. Therefore, I believe I would know if this bug was still there.

No problem for 24 hours.

**I believe 7.11.5 fixes this bug. Thanks @johannesjo!**
